demo/Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: ..
   specs:
-    primer_view_components (0.0.32)
+    primer_view_components (0.0.33)
```

Comment:
Should we be bumping this as part of the release process?

Reply:
it failed because of `mimemagic` :/